qa-test-app/src/main/java/com/mapbox/navigation/qa_test_app/view/InactiveRouteStylingActivity.kt

```diff
@@ -95,42 +88,63 @@ class InactiveRouteStylingActivity : AppCompatActivity() {
         MapboxRouteLineApi(options)
     }
 
-    override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
-        setContentView(binding.root)
-
-        initNavigation()
-        initStyle()
-        initListeners()
-    }
-
-    override fun onStop() {
-        super.onStop()
-        mapboxNavigation.unregisterRouteProgressObserver(replayProgressObserver)
```

Comment:
This is a bug. If you background the app it will unsubscribe. The subscribe happens in onCreate so the subscription is lost.